### ISSUE TITLE
composePeriods() sets incorrect values

### ISSUE THREAD

**Issue by dbvcode** [Reiryoku-Technologies/Mida#57]
I've just checked now, but with the latest Mida composePeriods() sets the high and low equal to the open.
It used to work well. Maybe the issue is since the MidaDecimal update?

**Comment by Vasile-Peste** [comment]
Can you post an example so I can reproduce this issue? Thank you for your patience!

**Comment by dbvcode** [comment]
```
  const firstTickDate = new Date();
  let ticksArray: MidaTick[] = [];
  marketWatcher.on("tick", (event) => {

      const tick = event.descriptor.tick as MidaTick;
      ticksArray.push(tick);

      const midaDate = new MidaDate(firstTickDate.valueOf());

      const lastCandle = composePeriods(ticksArray, midaDate, midaTF)[0];
      console.log(lastCandle.open, lastCandle.close, lastCandle.low, lastCandle.high);
  });
```

I think the problem is from `MidaDecimal.max` and `MidaDecimal.min` from `composePeriods()`

**Comment by Vasile-Peste** [comment]
I have reproduced the issue and already applied a fix which it's going to be delivered on the next release, thank you!

**Comment by Vasile-Peste** [comment]
Solved with [Mida 2022.2.0](https://github.com/Reiryoku-Technologies/Mida/releases/tag/v2022.2.0)
